src/respa.cpp

```diff
@@ -135,7 +138,7 @@ Respa::Respa(LAMMPS *lmp, int narg, char **arg) :
       }
       ++iarg;
     } else
-      error->all(FLERR, "Illegal run_style respa command");
+      error->all(FLERR, iarg + 1, "Unknown run_style respa keyword {}", arg[iarg]);
```

Comment:
Now I am confused here (sorry for jumping around), should this be `all(FLERR, iarg, ..., arg[iarg])`?

Trying to get a hold of the new `error->all` arguments.

Reply:
Got it, that's helpful. Thanks for the explanation. Hopefully I'll remember this next time it comes around.